src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,41 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
+  static deadAnimal() {
```

Comment:
Please fix the naming
```suggestion
  static filterOutDeadAnimals() {
```

```suggestion
  static removeEatenAnimals() {
```

etc